app/services/slack_notifier_service.rb

```diff
@@ -0,0 +1,34 @@
+class SlackNotifierService
+  CHANNEL = '#tariffs-etl'.freeze
+  USERNAME = 'Trade Tariff BE'.freeze
```

Comment:
What is BE? If this is backend, can it be changed to `Trade Tariff Backend` - that would remove the risk of a future developer not knowing what BE means at the cost of 5 letters.